src/ui/features/testnet-mode/TestModeDecoration/TestModeDecoration.tsx

```diff
@@ -0,0 +1,93 @@
+import { useId, useMemo } from 'react';
+import { useBodyStyle } from 'src/ui/components/Background/Background';
+import { templateData } from 'src/ui/shared/getPageTemplateName';
+import { KeyboardShortcut } from 'src/ui/components/KeyboardShortcut';
+import React from 'react';
+import { HStack } from 'src/ui/ui-kit/HStack/HStack';
+import { UIText } from 'src/ui/ui-kit/UIText';
+import { Toggle } from 'src/ui/ui-kit/Toggle';
+import { usePreferences } from '../../preferences';
+
+export function TestModeDecoration() {
```

Comment:
Not sure how big of a deal this is, but it does seem a bit odd when the extension pops up in a separate tab
<img width="507" alt="image" src="https://github.com/zeriontech/zerion-wallet-extension/assets/988849/a7731644-c8db-4e78-bf5c-43bf94206b8e">

Reply:
I see! we don't have bug report button enabled now, though

Reply:
ah, I see, thats my .env, nvm